experiments/scripts/subj-constants.sh

```diff
@@ -2,95 +2,103 @@
 rootDir=$1
 # The name of the subject you wish to add (e.g., xml_security).
 expName=(
-#  crystal
```

Comment:
Why are you making changes to this file? In the future, please double check your changes before your pull request. Especially for pull requests that are this big and difficult to fully digest.

Reply:
activemq and its paths need to be added to this file, but it looks like I commented out the paths for the other projects when I was testing the guidelines. Do you want me to commit the file again with all of the projects uncommented?